tests/test_cli.py

```diff
@@ -236,35 +261,125 @@ def test_studies_command() -> None:
+            if output_format is None or output_format == "table":
+                assert list(study.keys()) == expected_keys
+            else:
+                assert set(study.keys()) == set(expected_keys)
+
+        # Check study_name, direction, and n_trials for the first study.
+        assert studies[0]["name"] == study_1.study_name
+        if output_format is None or output_format == "table":
+            assert studies[0]["n_trials"] == "0"
+            assert eval(studies[0]["direction"]) == ("MINIMIZE",)
+        else:
+            assert studies[0]["n_trials"] == 0
+            assert studies[0]["direction"] == ["MINIMIZE"]
+
+        # Check study_name, direction, and n_trials for the second study.
+        assert studies[1]["name"] == study_2.study_name
+        if output_format is None or output_format == "table":
+            assert studies[1]["n_trials"] == "10"
+            assert eval(studies[1]["direction"]) == ("MINIMIZE", "MAXIMIZE")
+        else:
+            assert studies[1]["n_trials"] == 10
+            assert studies[1]["direction"] == ["MINIMIZE", "MAXIMIZE"]
+
+
+@pytest.mark.parametrize("output_format", (None, "table", "json", "yaml"))
+def test_studies_command_flatten(output_format: Optional[str]) -> None:
 
-        assert len(rows) == 6
-        assert tuple(get_row_elements(1)) == _Studies._study_list_header
+    with StorageSupplier("sqlite") as storage:
+        assert isinstance(storage, RDBStorage)
+        storage_url = str(storage.engine.url)
 
-        # Check study_name and n_trials for the first study.
-        elms = get_row_elements(3)
-        assert elms[0] == study_1.study_name
-        assert elms[2] == "0"
+        # First study.
+        study_1 = optuna.create_study(storage)
 
-        # Check study_name and n_trials for the second study.
-        elms = get_row_elements(4)
-        assert elms[0] == study_2.study_name
-        assert elms[2] == "10"
+        # Second study.
+        study_2 = optuna.create_study(
+            storage, study_name="study_2", directions=["minimize", "maximize"]
+        )
+        study_2.optimize(objective_func_multi_objective, n_trials=10)
+
+        # Run command.
+        command = ["optuna", "studies", "--storage", storage_url, "--flatten"]
+        if output_format is not None:
+            command += ["--format", output_format]
+
+        output = str(subprocess.check_output(command).decode().strip())
+        studies = _parse_output(output, output_format or "table")
+
+        if output_format is None or output_format == "table":
+            expected_keys_1 = [
+                "name",
+                "direction_0",
+                "direction_1",
+                "n_trials",
+                "datetime_start",
+            ]
+            expected_keys_2 = [
+                "name",
+                "direction_0",
+                "direction_1",
+                "n_trials",
+                "datetime_start",
+            ]
```

Comment:
`expected_keys_1` and `expected_keys_2` have exactly the same contents, and I guess we can simplify the code as follows:
```suggestion
            expected_keys_1 = expected_keys_2 = [
                "name",
                "direction_0",
                "direction_1",
                "n_trials",
                "datetime_start",
            ]
```